lib/contact_list.rb

```diff
@@ -50,6 +55,7 @@ def search(last_name)
     @list.select do |person|
       person["last_name"] == last_name
     end
+    p @list
   end
 
   def search_by(key, search_term)
```

Comment:
See comment above about the name of the argument `key`

Reply:
Also, the argument `search_term` is not used in this function so you can delete it

Reply:
here search_term is related to any word like for e.g. if I write Sarah, I will have all the info about this contact. Should I delete it too?